app/services/book_authors_service.rb

```diff
@@ -0,0 +1,13 @@
+class BookAuthorsService
+  def self.call(*args, &block)
+    new(*args, &block).call
+  end
+
+  def initialize(book)
+    @book = book
+  end
+
+  def call
+    @book.authors.map(&:id) if @book.authors.presence
```

Comment:
pluck is better in this case
https://rubyinrails.com/2014/06/05/rails-pluck-vs-select-map-collect/